test/ci/buildkite/test_runner.py

```diff
@@ -115,17 +111,17 @@ def main():
     elif (arguments.test_tf_python):
```

Comment:
The bazel test option can be removed

Reply:
Cool, maybe remove that and the GPU tests in a separate PR?